src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -698,7 +699,8 @@ prepareTxInfoRaw moneySource dstDistr = do
         coins = foldr1 unsafeAddCoin $ snd <$> dstDistr
     distr@(remaining, spendings) <- selectSrcAccounts coins notDstAccounts
     logDebug $ buildDistribution distr
-    bootEra <- gsIsBootstrapEra
+    epoch <- siEpoch <$> getCurrentSlotBlocking
```

Comment:
Hm, good question, I don't know, it should be discussed with wallet team. I see three options:
1. We can block and submit a transaction only after we are synchronized.
2. We can use `getCurrentSlotInaccurate`, then we don't need to block, but we may create invalid transaction.
3. We can simply tell user that we are not ready and do nothing.
I don't know who wallet behaves currently, but it has same options. It can block until we are synchronized. It can use current GState, but then it can create invalid transaction (if relevant parts of utxo changed). Or it can tell user that we are not ready.
It should be discussed with wallet team.